### PR TITLE
Project definition readme added

### PR DESCRIPTION
**Pull request by Senait-coding**
Hello Teammates :) I try to write our projects definition, feel free to  amend, comment ,even to change as a whole. thanks,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,62 @@
...
+- Slack
+
+## General Steps
+
+  1. [Create and Setup GitHub Repository](https://github.com/lauraramirez220612/TeamFourHYF)
+  2. [Write Project Definition](README.md)
+  3. [Write Backlog](planning/backlog.md)
+  4. [Make Wire-frame](planning/wireframe.md)
+  5. [Development Strategy](planning/development-strategy.md)
+  6. [Development](https://github.com/lauraramirez220612/TeamFourHYF/blob/main/index.html)
```
Comment (danielhalasz) [inline README.md:25]: maybe we could put this instead? what do you think?
https://lauraramirez220612.github.io/TeamFourHYF/

**Review by danielhalasz — APPROVED** [review]
looks really nice, my only questions is regarding the development link 🎉

File: README.md (hunk 1 above)
Comment (bringmetheaxx) [reply README.md:25]: I agree that it would be better with the GitHub Pages link.

**Review by bringmetheaxx — APPROVED** [review]

**Review by bringmetheaxx — CHANGES_REQUESTED** [review]
Please change it for the GitHub Pages url

File: README.md [hunk 2]
```diff
@@ -1,7 +1,62 @@
...
+- [Screenshot image 2](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165220_hackyourfuture.be.jpeg)
+- [Screenshot image 3](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165247_hackyourfuture.be.jpeg)
+
+- [Screenshot image 4](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165345_hackyourfuture.be.jpeg)
+
+- [screenshot image 5](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165443_hackyourfuture.be.jpeg)
+
+- [Screenshot image 6](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165512_hackyourfuture.be.jpeg)
+- [Screenshot image 7](https://github.com/lauraramirez220612/TeamFourHYF/blob/image/Image%20folder/Web%20capture_20-4-2021_165538_docs.google.com.jpeg)
+
```
Comment (Moamin-AbuEwaida) [inline README.md:41]: I clicked on the images links but they opened to a broken link  :thinking:

File: README.md [hunk 3]
```diff
@@ -1,7 +1,62 @@
...
+- [Laura-Ramirez](https://github.com/lauraramirez220612)
+
+- [TatsianaRud](https://github.com/TatsianaRud)
+
+- [Senait Berhane](https://github.com/Senait-coding)
+
+## Coach
+
+[Unmesh](https://github.com/unmeshvrije)
```
Comment (Moamin-AbuEwaida) [inline README.md:58]: i think it would be nice to add stephane as well

File: README.md [hunk 4]
```diff
@@ -1,7 +1,62 @@
...
 
 ## License & Copyright
 
-Licensed under the [MIT LICENSE](LICENSE)
\ No newline at end of file
+Licensed under the [MIT LICENSE](https://github.com/bringmetheaxx/TeamFour/blob/main/LICENSE)
```
Comment (Moamin-AbuEwaida) [inline README.md:62]: I think this license is for week1 project repo

**Review by Moamin-AbuEwaida — COMMENTED** [review]
I added some comment for your feedback

File: README.md (hunk 3 above)
Comment (danielhalasz) [reply README.md:58]: definitely

File: README.md (hunk 2 above)
Comment (danielhalasz) [reply README.md:41]: true, although the images are in the repo now..maybe because of the spaces in the file&dir names?

File: README.md (hunk 4 above)
Comment (danielhalasz) [reply README.md:62]: would you suggest to use a different one?

Comment (Moamin-AbuEwaida) [reply README.md:62]: We already have a license file for this repo so we can use it

Comment (Senait-coding) [reply README.md:62]: > I think this license is for week1 project repo
@Moamin-AbuEwaida  hello , the license part is I took yours as it is , I thought you have already updated it with the new link, anyway I will check and change it .thanks

File: README.md (hunk 3 above)
Comment (Senait-coding) [reply README.md:58]: ok  I will add him, thanks

File: README.md (hunk 2 above)
Comment (Senait-coding) [reply README.md:41]: thanks for the feedback I will check and fix it

File: README.md (hunk 3 above)
Comment (Senait-coding) [reply README.md:58]: @Moamin-AbuEwaida could you give me the link of Stéphane GitHub link please,  Unmesh is included in our repo so, I found his GitHub link  easily but couldn't find Stephane's  or else I  will check  in our previous  repo

Comment (snicoll) [reply README.md:58]: https://github.com/snicoll

**Review by danielhalasz — APPROVED** [review]
looks good

**Review by bringmetheaxx — APPROVED** [review]
Thank you, Senait, good work!!!

File: README.md (hunk 3 above)
Comment (Senait-coding) [reply README.md:58]: Thank you  @snicoll, I  already found your links from our previous Repo when Moamin got late to replay me :) and  I have amended the Readme accordingly.

**Comment by Moamin-AbuEwaida** [comment]
Thank you @Senait-coding on working on the document and following up on all the comments
Great output :+1:

**Pull request merged**